src/submap.h

```diff
@@ -91,6 +101,11 @@ class submap : public maptile_soa<SEEX, SEEY>    // TODO: Use private inheritanc
             ter[p.x][p.y] = terr;
         }
 
+        void set_all_ter( const ter_id &terr ) {
+            constexpr size_t block_size = SEEX * SEEY;
```

Comment:
This should be declared within the class itself, so it does not need to be declared within each function again and again.

Reply:
fixed